docs/getting-started.md

```diff
@@ -10,24 +10,16 @@ weight: 1
 
 Before you get started:
 
-- Install Git
```

Comment:
I would rather move all this to the "getting started for kudo developers" part than deleting it completely.

Reply:
Should we simply link to the [corresponding section](https://github.com/kudobuilder/kudo/blob/master/CONTRIBUTING.md#how-to-build-kudo-locally) of the contributing document?

Reply:
I updated `contributing.md`, making sure to include the info that I removed from the getting started guide.

I didn't add a link to the contributing guide, but I could if you think it would be useful =)